code/datums/supplypacks/mining.dm

```diff
@@ -31,3 +31,122 @@
 	containername = "\improper Planet Cracker suits"
 	access = access_mining
 
+
+/decl/hierarchy/supply_pack/mining/mining_cutter
```

Comment:
I've been asked and answered about these before. All of these guns had their supplypacks contained in their own files for ease of centralised rebalancing. It saves time and effort making tweaks to things

Reply:
We have also made it clear before that this is not how the system works and that you should stop doing it. Yet you persist.